webapp/packages/core-links/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@cloudbeaver/core-website",
```

Comment:
Why do we need to rename the package? I think both has the similar meaning

Reply:
it was about our website dbeaver.com, but now I can see it can be any links we have: website, github, social media, marketplaces etc. so left it as it is right now